components/brave_referrals/browser/brave_referrals_service.cc

```diff
@@ -1,7 +1,7 @@
-/* Copyright 2019 The Brave Authors. All rights reserved.
```

Comment:
You can add `(c)` only to prevent presubmit error. We don't want to udpate year.